doc/api/readline.md

```diff
@@ -403,8 +403,8 @@ a `'resize'` event on the `output` if or when the columns ever change
 
 ### Use of the `completer` Function
 
-When called, the `completer` function is provided the current line entered by
```

Comment:
Yes, `passed` is transitive so works.

`When called, the completer function is provided the current line entered by the user` === `When the completer function is called (by Node.js), the current line entered by the user is passed to it`.

I know we normally prefer the active voice, nothing against using that here, the only issue is that AIUI this is a callback function, so it `is called` by the Node.js readline code, the arguments are passed into it, and Node expects it to return an Array. So you can't say `Provide the function a line`, because the user doesn't do that, Node does.

You could maybe say something like:

>The `completer` function takes the current line entered by the user as an argument, and returns an Array with 2 entries:

I defer to you on what's clearer.